docs/introduction/setting-environment-variables.md

```diff
@@ -0,0 +1,86 @@
+# Setting environment variables
+
+This article describes how to set configure environment variables in different environment and even on some common used webservers.
+
+## Global configuration of environment variables
+
+Shopsys Framework as Symfony based application is using the same environment configuration as Symfony.
+
+We recommend using [`.env` files](https://symfony.com/doc/4.4/configuration.html#configuring-environment-variables-in-env-files) for configuring environment variables as it can be easily maintained. There is also advantage of different configuration through different application environments (suh as dev, test and prod).
```

Comment:
```suggestion
We recommend using [`.env` files](https://symfony.com/doc/4.4/configuration.html#configuring-environment-variables-in-env-files) for configuring environment variables as it can be easily maintained. There is also advantage of different configuration through different application environments (such as dev, test and prod).
```